modules/porous_flow/doc/content/modules/porous_flow/ates.md

```diff
@@ -0,0 +1,86 @@
+
+\begin{equation}
+R = \frac{\overline{h}_{\mathrm{p}} - h_{\mathrm{amb}}}{h_{\mathrm{i}} - h_{\mathrm{amb}}} 
+      \approx \frac{\overline{T}_{\mathrm{p}} - T_{\mathrm{amb}}}{T_{\mathrm{i}} - T_{\mathrm{amb}}} \ ,
+\end{equation}
+
+where $h$ denotes enthalpy and $T$ temperature.  The subscript "p" indicates "produced" (retrieved from the aquifer), "amb" indicates "ambient", and "i" indicates "injected".  For instance, if the ambient aquifer temperature is $T_{\mathrm{amb}} = 20^{\circ}$C, the injection temperature is $T_{i} = 150^{\circ}$C and $\reff=0.9$, then the average produced temperature is $\overline{T}_{\mathrm{p}} \approx 137^{\circ}$C.
+
+$R$ has been measured at ATES sites, and estimated using numerical modelling.  Numerical models can provide accurate estimates of $R$ as a function of operational parameters, such as the injection temperature and rate, and aquifer parameters such as the permeability, depth.  Such estimates are useful for rapid screening of potential ATES sites, indicating which sites might or might not be viable.
+
+The purpose of this page is to describe a MOOSE model of an ATES system, with the goal of predicting $R$, and present a few results.  For significantly more background discussion, discussion of the modelling approach, and results, the reader is referred to [!citet](sheldon2021).
+
+## Model setup
+
+A single-well ATES system with radial symmetry is explored here, in which hot water is injected and produced from the same well.  Hence, this may be simulated using "RZ" coordinates:
+
+!listing modules/porous_flow/examples/ates/ates.i block=Problem
+
+!listing modules/porous_flow/examples/ates/ates.i block=GlobalParams
+
+The geometry is shown in [ates_geometry_fig].
+
+!media porous_flow/ATES_geometry.png caption=Geometry on a radial slice of the ATES model.  id=ates_geometry_fig
+
+In this example, the mesh is created by using a combination of MOOSE MeshGenerators, however, for accurate prediction of $R$, readers are encouraged to use a more sophisticated approach, such as the one outlined in [!citet](sheldon2021).
+
+!listing modules/porous_flow/examples/ates/ates.i block=Mesh
+
+The core physics is handled by the [PorousFlowFullySaturated](PorousFlowFullySaturated.md) Action, and [Kuzmin-Turek stabilization](kt.md) is used to minimise numerical diffusion of the temperature front which impacts computation of $R$:
+
+!listing modules/porous_flow/examples/ates/ates.i block=PorousFlowFullySaturated
+
+The porepressure and temperature are fixed at the extremities of the model:
+
+!listing modules/porous_flow/examples/ates/ates.i start=[outer_boundary_porepressure] end=[inject_heat1]
+
+The injection and production of heat from the borehole are the most complicated aspect of this input file.  The boundary conditions corresonding to injection are fixed temperature and constant rate of fluid flux:
+
+!listing modules/porous_flow/examples/ates/ates.i start=[inject_heat1] end=[produce_heat1]
```

Comment:
Minor, but the '1' at the end of each of these sink names could be removed

Reply:
Done